LEGO1/mxvideomanager.h

```diff
@@ -26,13 +26,17 @@ class MxVideoManager : public MxMediaManager
 
   inline MxVideoParam& GetVideoParam() { return this->m_videoParam; }
   inline LPDIRECTDRAW GetDirectDraw() { return this->m_pDirectDraw; }
+  inline MxDisplaySurface *GetDisplaySurface() { return this->m_displaySurface; }
+  inline void SetUnkE4(MxBool p_value) { this->m_unke4 = p_value; }
 private:
   MxVideoParam m_videoParam;
   LPDIRECTDRAW m_pDirectDraw;
   LPDIRECTDRAWSURFACE m_pDDSurface;
   MxDisplaySurface *m_displaySurface;
   MxRegion *m_region;
   MxBool m_unk60;
+  undefined m_pad61[84];
```

Comment:
Are you sure this is part of `MxVideoManager` and not `LegoVideoManager`? Regardless, we have to update the class size and assert.

Reply:
I think you're right, thanks. My bad, this was just an erroneous assumption on my part.